tests/postgres/grammar/test_select_grammar.py

```diff
@@ -69,6 +69,12 @@ def can_compile_with_order_by(self):
         """
         return """SELECT "users"."username" FROM "users" ORDER BY "users"."age" DESC"""
 
+    def can_compile_with_multiple_order_by(self):
+        """
+        self.builder.select('username').order_by('age', 'desc').order_by('name').to_sql()
+        """
+        return """SELECT "users"."username" FROM "users" ORDER BY "users"."age" DESC, "users"."name" ASC"""
```

Comment:
So this is a little tricky but how the select grammars tests work (and really how all test classes will work in the future when things aren't as fluid as they have been) Is that there is a `BaseTestCaseSelectGrammar` (see line 8) that has the test cases. those are inherited to the child test classes and call the related test methods to make the right assertions per grammar.

So for example a test case inside the `BaseTestCaseSelectGrammar` has a test case like `test_can_compile_with_multiple_order_by` which calls this method here (`can_compile_with_multiple_order_by` without the `test_` part). This allows the abstraction of tests that are the same to trickle down to the child test classes. So adding a test in 1 place basically calls tests for every grammar. Keeping it DRY

What I'm trying to say is that this test is not actually running without the `test_can_compile_with_multiple_order_by` inside the `BaseTestCaseSelectGrammar`